src/reanimated2/core.ts

```diff
@@ -45,10 +45,11 @@ export const isReanimated3 = () => true;
 export const isConfigured = isReanimated3;
 
 // this is for web implementation
-global._WORKLET = false;
-global._log = function (s: string) {
-  console.log(s);
-};
+if (shouldBeUseWeb()) {
+  global._WORKLET = false;
+  global._log = console.log;
+  global._getAnimationTimestamp = () => performance.now();
```

Comment:
We can't assign `performance.now` directly here because we get `TypeError: Illegal invocation` when calling `_getAnimationTimestamp`. Another solution would be to assign `performance.now.bind(performance)` (it works in runtime) but this breaks Jest tests.